lib/awful/client.lua

```diff
@@ -999,11 +999,22 @@ function client.is_transient_for(c, c2)
 end
 
 -- Register standards signals
+
+--- The last geometry when client was floating.
+-- @signal property::floating_geometry
 capi.client.add_signal("property::floating_geometry")
+
 capi.client.add_signal("property::floating")
+
 capi.client.add_signal("property::dockable")
+
+--- The client marked signal.
+-- @signal .marked
 capi.client.add_signal("marked")
-capi.client.add_signal("unmarked")
+
+--- The client unmarked signal.
+-- @signal unmarked
+capi.client.add_signal(".unmarked")
```

Comment:
I don't think you wanted to add the `.` here. Perhaps better the line before?
(Also, why the heck do we have marked and unmarked and not just property::marked...)